### PR TITLE
Add functional tests for firefox/desktop/ pages

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by alexgibson** [comment]
@davehunt r?

File: tests/functional/firefox/desktop/test_customize.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        sections[i].click_next()
+    assert sections[0].is_displayed
+
+
+@pytest.mark.nondestructive
+def test_theme_buttons(base_url, selenium):
+    page = CustomizePage(base_url, selenium).open()
+    themes = page.themes
+    assert themes[3].is_selected
+    assert themes[3].is_image_displayed
```
Comment (davehunt) [inline tests/functional/firefox/desktop/test_customize.py]: If you want to access the last item in a list you can do so without knowing the total number of items using `themes[-1]`

File: tests/pages/firefox/desktop/customize.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+
+
+class CustomizePage(FirefoxBasePage):
```
Comment (davehunt) [inline tests/pages/firefox/desktop/customize.py]: Should this extend `FirefoxDesktopBasePage`?

File: tests/pages/firefox/desktop/desktop.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+from pages.regions.download_button import DownloadButton
+
+
+class DesktopPage(FirefoxBasePage):
```
Comment (davehunt) [inline tests/pages/firefox/desktop/desktop.py]: Should this extend `FirefoxDesktopBasePage`?

**Comment by davehunt** [comment]
r+ wc

**Comment by alexgibson** [comment]
Thanks @davehunt

**Pull request merged**